website/build_site.sh

```diff
@@ -22,7 +22,8 @@ $pythonCmd -m pip install -e "$MLOS_ROOT/source/Mlos.Python/"
 $pythonCmd -m pip install pyyaml
 $pythonCmd -m pip install \
     $("$MLOS_ROOT/scripts/parse-pip-requirements-from-environment-yaml.py" "$MLOS_ROOT/source/Mlos.Notebooks/environment.yml")
-$pythonCmd -m pip install jupyter nbconvert
+# FIXME: nbconvert 6.0.1 had an error.
+$pythonCmd -m pip install jupyter nbconvert==5.6.1
```

Comment:
Without this we currently get an error if pip selects a new version (e.g. 6.0.1):

```
jinja2.exceptions.TemplateNotFound: index.md.j2
```

Reply:
Will take that up in https://github.com/microsoft/MLOS/issues/68